src/app/(homepage)/HomepageTabs.js

```diff
@@ -12,30 +12,30 @@ import { useTheme } from "@mui/material/styles";
 import useMediaQuery from "@mui/material/useMediaQuery";
 import SchoolStats from "@/app/(homepage)/SchoolStats";
 
-function HomepageTabsHeader({ sx }) {
-  return (
-    <Box {...sx}>
-      <Typography variant="subheading">
-        Effortless tracking of every important detail
-      </Typography>
-      <Typography mt={2} variant="h2">
-        Your central hub for key school details
-      </Typography>
-      <Typography mt={2} variant="body1" color="text.secondary">
-        You shouldn’t have your work cut out finding and syncing holidays,
-        birthdays and contacts. Once you&apos;re on board, you&apos;ll have access to
-        crucial information like school holiday schedules, classmates&apos;
-        birthdays, and parents&apos; contact details at your fingertips.
-      </Typography>
-    </Box>
-  );
+
+const SchoolDetail = {
```

Comment:
Important convention in React is that CapitalCase are only components. This should be UPPER_CASE as a constant data.